### ISSUE TITLE
Strange behaviour on iOS 13

### ISSUE THREAD

**Issue by pdefuns** [mintware-de/flutter_barcode_reader#151]
It looks like a special overlay and scans the underlying page first. Then switches to camera..

**Comment by lookfirst** [comment]
@YAPPDEF yea... will likely switch to this one... swift based, more modern and developer is responsive... https://github.com/AmolGangadhare/flutter_barcode_scanner

**Comment by lookfirst** [comment]
Just following up... yep, replaced this one with the one above and it is much better. Took me a bit longer cause I didn't setup my flutter project originally with swift, so I had to redo my ios folder... but glad I did... the above works great on ios13 and has a nice animation for the scanning line.

**Comment by pdefuns** [comment]
@lookfirst thanks! I'll check it out

**Comment by appditto** [comment]
This is because the default UIPresentationStyle has changed in iOS 13.

```
if (@available(iOS 13.0, *)) {
    [scannerViewController setModalPresentationStyle:UIModalPresentationFullScreen];
}
```

This should fix it, we made the change in our fork of this plugin here: https://github.com/bbedward/flutter_barcode_reader/commit/98869b04a546640f9652da1a6c5e7f1c0b209ba0
